AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockSafeContentFilterTest.xml

```diff
@@ -28,6 +28,10 @@
             <actionGroup ref="resetAdminDataGridToDefaultView" stepKey="resetAdminDataGridToDefaultView"/>
             <actionGroup ref="logout" stepKey="logout"/>
         </after>
+        
+        <actionGroup ref="AdminSearchImagesOnModalActionGroup" stepKey="searchForPear">
```

Comment:
Can we update the `stepKey` here to name the search appropriate, like e.g. `searchForCars`?
```suggestion
        <actionGroup ref="AdminSearchImagesOnModalActionGroup" stepKey="searchForCars">
```